modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,146 @@
+= Configure Role-based Access Control
```

Comment:
```suggestion
= Configure Role-Based Access Control
```
@kbatuigas checking that you're OK with this!

Reply:
@Feediver1 thoughts on capitalization here?